geomesa-kafka/geomesa-kafka-utils/src/main/scala/org/locationtech/geomesa/kafka/KafkaConsumerVersions.scala

```diff
@@ -14,7 +14,7 @@ import org.apache.kafka.clients.consumer.{Consumer, ConsumerRebalanceListener}
 import org.apache.kafka.common.TopicPartition
 
 /**
-  * Reflection wrapper for KafkaConsumer methods between kafka versions 0.9 and 0.10
+  * Reflection wrapper for KafkaConsumer methods between kafka versions 0.9, 0.10, 1.0, 1.1, and 2.0
```

Comment:
This seems to suggest that Kafka 0.9 - 2.0 are supported.

Reply:
yeah, that is correct